<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GdalToolsWidget</class>
 <widget class="QWidget" name="GdalToolsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>342</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Build overviews (Pyramids)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="batchCheck">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Batch mode (for processing whole directory)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Input file</string>
       </property>
       <property name="buddy">
        <cstring>inSelector</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="GdalToolsInOutSelector" name="inSelector" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cleanCheck">
     <property name="toolTip">
      <string>Remove all overviews.</string>
     </property>
     <property name="text">
      <string>Clean</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsRasterPyramidsOptionsWidget" name="mPyramidOptionsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GdalToolsInOutSelector</class>
   <extends>QWidget</extends>
   <header>.inOutSelector</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsRasterPyramidsOptionsWidget</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
